bin/tictactoe

```diff
@@ -1,3 +1,42 @@
-#!/usr/bin/env ruby
-
 require_relative '../config/environment'
+
```

Comment:
@Lani1321 @klovae a great design pattern would have you abstract line 2 - line 42 into a CLI class in the lib folder and then you would just call a new instance of that class. `CLI.new.new_game`